site/_posts/2017-10-24-api-contributors-guide.md

```diff
@@ -0,0 +1,252 @@
+---
+title: API Contributor's Guide
+author: jntullo
+date: 2017-10-24
+tags: collaboration developers tutorials api
+comments: true
+published: true
+---
+
+The API has been growing quickly thanks to our many contributors. In an effort to help new contributors get up to speed quickly, it was about time for a blog post to explain the process of adding in a new collection and subcollection, as well as provide some examples for commonly asked questions.
+
+The [api.yml](https://github.com/ManageIQ/manageiq-api/blob/4811ec9fcea2424a585560a761e05d56c53535f8/config/api.yml) is where a lot of the magic :sparkles: happens. The best way to explain it is through an example, so let's walk through adding in a new collection and subcollection.
+
+We'll keep the example generic through the creation of a `Coffee` API. Because who doesn't need more :coffee: in their life?
+
+## Defining a new collection with basic CRUD
+The new `Coffee` collection will need to be defined under [collections](https://github.com/ManageIQ/manageiq-api/blob/4811ec9fcea2424a585560a761e05d56c53535f8/config/api.yml#L37) in the `api.yml`. Note that all of the collections are in alphabetical order. 
```

Comment:
You just reminded me of [this](https://tools.ietf.org/html/rfc2324) 😆

Reply:
@imtayadeway hah! 😆